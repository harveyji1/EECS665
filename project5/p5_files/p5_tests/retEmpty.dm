fn: (a: int, b: int) int{
    hello: int;
    return;
}

main: () void{
    fn(3, 3);
}