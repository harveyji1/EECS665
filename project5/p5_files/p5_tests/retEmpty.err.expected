FATAL [3,5]-[3,11]: Missing return value
Type Analysis Failed